documentation/homework/homeworkweek1/App.js

```diff
@@ -0,0 +1,76 @@
+'use strict';
+
+const ListItem = props => {
+  const { description, deadline } = props.listInfo;
```

Comment:
Nice using of stateless component.

You have made a re-usable Component that share use for both the dynamic and the static lists, but used it only in the static 🤔